packages/EasyErrorHandler/src/Bridge/Symfony/Resources/config/services.php

```diff
@@ -56,6 +58,9 @@
             'template' => '@EasyErrorHandlerSymfony/Collector/error_handler_collector.html.twig',
         ]);
 
+    // BugsnagIgnoreExceptionsResolver
+    $services->set(BugsnagIgnoreExceptionsResolverInterface::class, BugsnagIgnoreExceptionsResolver::class);
```

Comment:
What about configuration, similar to `bugsnagIgnoredExceptions`? So we can see `bugsnagIgnoredExceptions` and `bugsnagIgnoreExceptionsResolver` options in one place and explicitly see all the ignore-related configuration?